cypress/integration/QuestionForm.spec.js

```diff
@@ -188,96 +189,55 @@ context('QuestionForm', () => {
 
     cy.get('button').click()
 
-    cy.get('li').contains('Name des Vereins')
-    cy.get('p').contains('Carolin')
-
-    cy.get('li').contains('Stadt')
-    cy.get('p').contains('Hamburg')
-
-    cy.get('li').contains('Zweck')
-    cy.get('p').contains('testen')
-
-    cy.get('li').contains('Wer soll Mitglied')
-    cy.get('p').contains('Unternehmen')
-
-    cy.get('li').contains('Mitglieder einen Mitgliedsantrag')
-    cy.get('p').contains('schriftlich und email')
-
-    cy.get('li').contains('Aufnahmegebühr')
-    cy.get('p').contains('ja')
-
-    cy.get('li').contains('Belange soll die Mitgliedsversammlung')
-    cy.get('p').contains('Vorstandsentlastung und Haushaltsplan')
-
-    cy.get('li').contains('Einladung zur Mitgliederversammlung')
-    cy.get('p').contains('per eMail')
-
-    cy.get('li').contains('Tage vorher soll zur Mitgliederversammlung')
-    cy.get('p').contains('21')
-
-    cy.get('li').contains('Mitgliederversammlung beschlussfähig')
-    cy.get('p').contains('Immer')
-
-    cy.get('li').contains('häufig soll eine Mitgliederversammlung')
-    cy.get('p').contains('monatlich')
-
-    cy.get('li').contains(
-      'Mehrheit soll die Mitgliederversammlung ihre Beschlüsse'
-    )
-    cy.get('p').contains('einfache')
-
-    cy.get('li').contains('Vorstand zusammengesetzt')
-    cy.get('p').contains('zwei Vorsitzende und Schatzmeister')
-
-    cy.get('li').contains('Verein nach außen vertreten')
-    cy.get('p').contains('Vorstandsmehrheit')
-
-    cy.get('li').contains('Zeitraum soll der Vorstand')
-    cy.get('p').contains('drei Jahre')
-
-    cy.get('li').contains('Wiederwahl des Vorstandes')
-    cy.get('p').contains('Ja')
-
-    cy.get('li').contains('zur Vorstandssitzung geladen')
-    cy.get('p').contains('schriftlich')
-
-    cy.get('li').contains('Tage vorher soll zu einer Vorstandssitzung')
-    cy.get('p').contains('30')
-
-    cy.get('li').contains('Mitteilung einer Tagesordnung')
-    cy.get('p').contains('Nein')
-
-    cy.get('li').contains('Arbeitsausschüsse')
-    cy.get('p').contains('Ja')
-
-    cy.get('li').contains(
-      'Mitgliederversammlung über die Auflösung des Vereins'
+    cy.get('#nameAndLocation').contains('Carolin')
+    cy.get('#businessYear').contains('Kalenderjahr')
+    cy.get('#clubPurpose').contains('testen')
```

Comment:
Does this cypress test work as expected? You use "get" to get each element, but you don't use "should" to specify an expected result. I thought you need something like "should('exist')" or "should('have.length', 1)".

Reply:
Yes, it works as expected